src/Microsoft.Diagnostics.Tracing.EventSource.Redist/dir.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <PackageVersion>2.0.1</PackageVersion>
+    <PackageVersion>2.1.0</PackageVersion>
```

Comment:
Looks like this one behaves different than the rest of CoreFx.  IIRC @brianrob put this here to piggyback on corefx infra.  You might want to chat with him about this change (and what should be done for 2.1).